### PR TITLE
107 Broken Members Dropdown

### LINKED ISSUE #107: Broken Members Dropdown
## Description

When I hover over 'members' in the header, the dropdown's styling is broken - years are side-by-side and stacked rather than vertical.

## Expected behavior

Years are ordered vertically with the most recent year at the top. No stacking.

## To Reproduce

1. Go to any page
1. Hover over 'members' in navbar
1. If there is more than 1 year in Strapi, see the years stacked horizontally.

### PR DESCRIPTION
**Pull request by kk7-py**
## Context

- Fixed broken dropdown menu, years are now aligned vertically
- Years are also sorted in a descending order

Closes #107 

## What Changed?
- Removed and replaced sorting logic by converting years into a number and sorting the value in a descending order
- Fixed dropdown menu by aligning the years vertically.
- Centred the years in dropdown menu and slightly adjusted the padding to fit the years better. 

## How To Review

- Hover over the members heading on the website to see the hidden dropdown menu, and check if the years are aligned vertically and in descending order. 

- Checkout 
**next/components/header/header.tsx**
to view the changes. 
- Review implemented sorting logic and styles

### CODE REVIEW

File: next/components/header/header.tsx [hunk 1]
```diff
@@ -40,16 +47,23 @@ export default async function Header() {
...
-          <div className="group-hover:flex hidden absolute top-full bg-white p-2 rounded-md items-center text-b-dark-blue">
-            {/* @ts-ignore */}
-            {data.members.map(({ CommitteeYear }) => (
+          <div
+            className="group-hover:flex hidden absolute top-full bg-white p-2 rounded-md items-center text-b-dark-blue"
+            style={{
+              flexDirection: "column",       // Stack the years vertically
+              alignItems: "center",          // Center the text horizontally
+              padding: "0.25rem 0.5rem",     // Adjust padding to fit the text better
+            }}
```
Comment (Oculux314) [inline next/components/header/header.tsx]: Convert these three to tailwind classes ^

https://tailwindcss.com/docs/flex-direction
https://tailwindcss.com/docs/padding

**Review by Oculux314 — COMMENTED** [review]
Concise bugfix, detailed description, great naming and commit conventions, AND implementing additional error checking on your own initiative? Dang this is a really great PR. See one place where I think you can use tailwind rather than inline CSS (references provided!) - fix that and I'm happy to merge!

(P.s. I altered some confusing code from 2023 that was tangentially-related to your PR so do a `git pull` on this branch before changing anything 😄)

**Review by Oculux314 — APPROVED** [review]
Lgtm thanks for those changes :)

**Review by YvonneLiew — APPROVED** [review]
lgtm

**Comment by Oculux314** [comment]
@kk7-py yo! Two approvals means you've got the go-ahead to merge this into main, GJ :D

**Pull request merged**